src/test/ui/pattern/bindings-after-at/copy-and-move-mixed.rs

```diff
@@ -8,10 +8,6 @@ struct C;
 struct NC<A, B>(A, B);
 
 fn main() {
-    let a @ NC(b, c) = NC(C, C);
```

Comment:
you lost this test

Reply:
it's a passing test. I wanted to add it pass. (part of TODO)

Reply:
still, it doesn't hurt to keep it here to show that this passes in contrast to the non-passing version below